### DISCUSSION TITLE
Add support for seconds

### DISCUSSION THREAD

**Discussion by RicardoFavas** [xrutayisire/react-js-cron#53] [Ideas]
![image](https://user-images.githubusercontent.com/26768182/227233129-c1b8bf7c-72d7-4f55-ab85-ea0cfc68f95e.png)

**Comment by xrutayisire** [comment]
Hello,

It's normal, UNIX cron syntax only support 5 values, here you have 6. 
See: https://en.wikipedia.org/wiki/Cron

6 values mean it's a Quartz syntax not supported at the moment.
See: https://github.com/xrutayisire/react-js-cron/issues/46
